<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="Dhinu" failures="4" tests="4" name="Test" time="8.26" errors="0" timestamp="2024-07-13T19:47:55 IST">
  <testcase classname="org.calculator.Calculator_Project.CalculatorTests" name="calculate" time="1.595">
    <failure type="java.lang.AssertionError" message="expected [11] but found [5028]">
      <![CDATA[java.lang.AssertionError: expected [11] but found [5028]
at org.calculator.Calculator_Project.CalculatorTests.calculate(CalculatorTests.java:34)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- calculate -->
  <testcase classname="org.calculator.Calculator_Project.CalculatorTests" name="calculate" time="1.978">
    <failure type="java.lang.AssertionError" message="expected [17] but found [-245]">
      <![CDATA[java.lang.AssertionError: expected [17] but found [-245]
at org.calculator.Calculator_Project.CalculatorTests.calculate(CalculatorTests.java:34)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- calculate -->
  <testcase classname="org.calculator.Calculator_Project.CalculatorTests" name="calculate" time="1.965">
    <failure type="java.lang.AssertionError" message="expected [236] but found [5028]">
      <![CDATA[java.lang.AssertionError: expected [236] but found [5028]
at org.calculator.Calculator_Project.CalculatorTests.calculate(CalculatorTests.java:34)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- calculate -->
  <testcase classname="org.calculator.Calculator_Project.CalculatorTests" name="calculate" time="2.281">
    <failure type="java.lang.AssertionError" message="expected [1045] but found [-245]">
      <![CDATA[java.lang.AssertionError: expected [1045] but found [-245]
at org.calculator.Calculator_Project.CalculatorTests.calculate(CalculatorTests.java:34)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- calculate -->
  <testcase classname="org.calculator.Calculator_Project.CalculatorTests" name="testSubtraction" time="0.0">
    <ignored/>
  </testcase> <!-- testSubtraction -->
  <testcase classname="org.calculator.Calculator_Project.CalculatorTests" name="testMultiplication" time="0.0">
    <ignored/>
  </testcase> <!-- testMultiplication -->
  <testcase classname="org.calculator.Calculator_Project.CalculatorTests" name="testAddition" time="0.0">
    <ignored/>
  </testcase> <!-- testAddition -->
  <testcase classname="org.calculator.Calculator_Project.CalculatorTests" name="testDivision" time="0.0">
    <ignored/>
  </testcase> <!-- testDivision -->
</testsuite> <!-- Test -->
